source/installation.rst

```diff
@@ -6,7 +6,7 @@ Installation
 The OnDemand host machine needs to be setup *similarly* to a login node. This
 means that it will need:
 
-- RedHat/CentOS 7+
+- RedHat/CentOS 7+ or Ubuntu 18.04+
```

Comment:
Is this enough for #710? IDK where else we call out the operating systems.

Reply:
I think that's good.  Though I'd recommend limiting to 7-8 and Ubuntu 18.04 and 20.04 because we don't support el9 or 22.04 with 2.0 releases.

Reply:
updated.